src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimize.cpp

```diff
@@ -198,17 +198,27 @@ bool QMCFixedSampleLinearOptimize::run()
 #ifdef HAVE_LMY_ENGINE
   if (doHybrid)
   {
+#if !defined(QMC_COMPLEX)
     app_log() << "Doing hybrid run" << std::endl;
     return hybrid_run();
+#else
+APP_ABORT(" Error: Hybrid method does not work with QMC_COMPLEX=1. \n");
```

Comment:
The communicator is accessible in this class. Use `myComm->barrier_and_abort()` to have clean print.